### DISCUSSION TITLE
Need clarification on Success Criterion 2.5.3: Label in Name

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#2731] [Q&A] [answered]
Hi WCAG Community,

I have a question related to '2.5.3: Label in Name'. In the example(Please look into the screenshot)
The accessible name is given as: Mute All
Visible label is:  Mute All People can unmute themselves

Here the 'People can unmute themselves' text is not a part of an accessible name. 

I think it's fine since 'Mute All' text is more relevant here and by saying this text, speech input user can activate the control.  

**So my question is:**
1. Can we consider it as a pass
or 
2. It is still a failure of the Label in name checkpoint since 'People can unmute themselves' is not a part of accessible name(aria-label)

**As per WCAG:**
In order for the label text and accessible name to be matched, it is first necessary to determine which text on the screen should be considered a label for any given control. There are often multiple text strings in a user interface that may be relevant to a control. However, there are reasons why it is best to conservatively interpret the label as being only the text in close proximity.
https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html
![Label in name](https://user-images.githubusercontent.com/115781693/195801646-d942d2a4-73ff-4d27-9903-a878b4d12cb8.png)

**Comment by JAWS-test** [comment]
Pass becauce name is "Mute all", description is "People can..."

See also https://github.com/w3c/wcag/issues/2276

**Comment by patrickhlauke** [comment, marked as answer]
Agree, the apparent label here is just "Mute all" (reasonably, a voice user would likely not try to activate that button by reading out all the lengthy text there either).

I would be concerned though that the additional description there is not exposed in any other way to screen reader users, as it's overridden by the short `aria-label`? that'd be something to consider under 1.3.1 Info and Relationships or similar...

**Comment by JAWS-test** [comment]
Please use aria-describedby to refer from the button to the description "People can..."

**Comment by AccessibilityEnthusiastic** [comment]
Thank you!! I am closing the issue.

**Reply by bruce-usab** [reply]
@AccessibilityEnthusiastic please "Mark as answer" as you deem appropriate in the this discussion thread.

**Comment by patrickhlauke** [comment]
(retrospectively turned this into a discussion, rather than an issue, as it's asking a question/clarification of how to apply an SC, rather than an issue with the SC itself)

**Reply by bruce-usab** [reply]
Nice.  I will see I can do that the next time I am first to reply to an Issue that seems like it asking a question or for clarification.
